### PR TITLE
Main Game logic

### PR DESCRIPTION
**Pull request by KerronKing**
In this project, we:

- Created the skeleton of the page with HTML and CSS.
- Added a form to collect player data.
- Created an array to hold player information and the function that renders the game board.
- Added event listeners to update the game board with player symbols.
- Added the logic to check if a game is won/lost or drawn.
- Added the ability for the user to reset the game at any point.
- Edited overall styling.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
+// Global Array to store player objects
```
Comment (bellom) [inline app.js]: **Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face:**
**_Do this in all your files...._**

**Comment by KerronKing** [comment]
Good night Oluwaseun. All comments have been removed.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
+let players = [];
```
Comment (bellom) [inline app.js]: **We can call `players` anywhere in this code, we told not to use global code right ?**

It doesn't use `global` code - encapsulate logic inside modules or factories ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 
#### Awesome job so far guys!! :+1: 
#### You have done wonderfully well, you passed almost all checklist for this project :+1: 
1. we have this little bug; player with sign X already won, but no sign of winning the game, as you can see below;
image

**Kindly re-submit for review after making all necessary changes :slightly_smiling_face: Thumps -up await you....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by KerronKing** [comment]
Good Day Oluwaseun. All suggested changes were made. We look forward to a fresh review.

Thanks!

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Goodluck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**